packages/pubsub/package.json

```diff
@@ -51,9 +51,12 @@
   ],
   "dependencies": {
     "@google-cloud/common": "^0.1.0",
+    "arguejs": "^0.2.3",
     "arrify": "^1.0.0",
     "extend": "^3.0.0",
+    "google-gax": "^0.6.0",
     "is": "^3.0.1",
+    "lodash": "^4.15.0",
```

Comment:
Can we use the [`lodash.union`](https://npmjs.com/lodash.union) dependency instead?